src/Sulu/Bundle/AdminBundle/Tests/Unit/FormMetadata/FormXmlLoaderTest.php

```diff
@@ -111,139 +117,149 @@ public function testLoadFormWithLocalization()
+        /** @var LocalizedFormMetadataCollection $formMetadataCollection */
+        $formMetadataCollection = $this->loader->load($this->getFormDirectory() . 'form_with_schema.xml');
+
+        /** @var FormMetadata $formMetadataEn */
+        $formMetadata = $formMetadataCollection->get('en');
 
         $this->assertInstanceOf(FormMetadata::class, $formMetadata);
 
-        $this->assertCount(4, $formMetadata->getProperties());
+        $this->assertCount(3, $formMetadata->getItems());
 
-        $this->assertEquals('first', $formMetadata->getProperties()['first']->getName());
-        $this->assertEquals('second', $formMetadata->getProperties()['second']->getName());
+        $this->assertEquals('first', $formMetadata->getItems()['first']->getName());
+        $this->assertEquals('second', $formMetadata->getItems()['second']->getName());
 
         $this->assertEquals(
             [
                 'required' => [],
-                'anyOf' => [
+                'allOf' => [
                     [
-                        'required' => [],
-                        'properties' => [
-                            'first' => [
-                                'name' => 'first',
-                                'const' => 1,
-                            ],
+                        'required' => [
+                            'first',
+                            'third',
                         ],
                     ],
                     [
                         'required' => [],
-                        'properties' => [
-                            'second' => [
-                                'name' => 'second',
-                                'const' => 2,
+                        'anyOf' => [
+                            [
+                                'required' => [],
+                                'properties' => [
+                                    'first' => [
+                                        'name' => 'first',
+                                        'const' => 1,
+                                    ],
+                                ],
+                            ],
+                            [
+                                'required' => [],
+                                'properties' => [
+                                    'second' => [
+                                        'name' => 'second',
+                                        'const' => 2,
+                                    ],
+                                ],
                             ],
                         ],
-                    ],
-                ],
-                'allOf' => [
-                    [
-                        'required' => [],
-                        'properties' => [
-                            'first' => [
-                                'name' => 'first',
-                                'const' => 1,
+                        'allOf' => [
```

Comment:
Are you sure these changes are ok? Seems like it behaves differently as before...

Reply:
The merging of schema information happened previously in the FormMetadataProvider and now it's already done in the FormXmlLoader and so the FormXmlLoader returns the merged schema information. 
As the properties first and third are mandatory it seems correct to me that they are contained in the schema.